### PR TITLE
Verifying Amazon content - corrections.

### PR DESCRIPTION
**Pull request by mihaitusa**
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
@oritstone Please review it. Is ready for merge.

File: content/io/cloudslang/cloud_provider/amazon_aws/list_servers.sl [hunk 1]
```diff
@@ -6,17 +6,20 @@
...
-#      - proxyPort - optional - the proxy server port used to access the provider services
-#      - delimiter - optional - the delimiter used in result list
+# Inputs:
+#   - provider - the cloud provider on which the instance is - Default: 'amazon'
+#   - endpoint - the endpoint to which first request will be sent - Default: 'https://ec2.amazonaws.com'
+#   - identity - optional - the Amazon Access Key ID
+#   - credential - optional - the Amazon Secret Access Key that correspond to the Amazon Access Key ID
+#   - region - optional - the region where the servers (instances) are. list_regions operation can be used in order
+#                         to get all regions
+#                       - Default: 'us-east-1'
```
Comment (oritstone) [inline content/io/cloudslang/cloud_provider/amazon_aws/list_servers.sl]: No need for new line for the default value

File: content/io/cloudslang/cloud_provider/amazon_aws/start_server.sl [hunk 2]
```diff
@@ -6,18 +6,20 @@
...
-#      - proxyHost - optional - the proxy server used to access the provider services
-#      - proxyPort - optional - the proxy server port used to access the provider services
+# Inputs:
+#   - provider - the cloud provider on which the instance is - Default: 'amazon'
+#   - endpoint - the endpoint to which first request will be sent - Default: 'https://ec2.amazonaws.com'
+#   - identity - optional - the Amazon Access Key ID
+#   - credential - optional - the Amazon Secret Access Key that correspond to the Amazon Access Key ID
+#   - region - optional - the region where the server (instance) to be started can be found. list_regions operation
+#                         can be used in order to get all regions
+#                       - Default: 'us-east-1'
```
Comment (oritstone) [inline content/io/cloudslang/cloud_provider/amazon_aws/start_server.sl]: No need for new line for the default value

File: content/io/cloudslang/cloud_provider/amazon_aws/stop_server.sl [hunk 3]
```diff
@@ -6,18 +6,20 @@
...
-#      - proxyHost - optional - the proxy server used to access the provider services
-#      - proxyPort - optional - the proxy server port used to access the provider services
+# Inputs:
+#   - provider - the cloud provider on which the instance is - Default: 'amazon'
+#   - endpoint - the endpoint to which first request will be sent - Default: 'https://ec2.amazonaws.com'
+#   - identity - optional - the Amazon Access Key ID
+#   - credential - optional - the Amazon Secret Access Key that correspond to the Amazon Access Key ID
+#   - region - optional - the region where the server (instance) to be stopped can be found
+#                         list_regions operation can be used in order to get all regions
+#                       - Default: 'us-east-1'
```
Comment (oritstone) [inline content/io/cloudslang/cloud_provider/amazon_aws/stop_server.sl]: No need for new line for the default value

**Pull request merged**